app/models/pxe_server.rb

```diff
@@ -213,4 +213,21 @@ def ensure_menu_list(menu_list)
   def self.display_name(number = 1)
     n_('PXE Server', 'PXE Servers', number)
   end
+
+  def self.verify_depot_settings_queue(userid, options)
+    task_opts = {
+      :action => "Verify PXE Server Credentials",
+      :userid => userid
+    }
+
+    queue_opts = {
+      :class_name  => name,
+      :method_name => "verify_depot_settings",
```

Comment:
Where is this method?  I don't see it on this class.

Reply:
Oh I see it FileDepotMixin.  In my opionion, the `*_queue` methods should live with their counterpart, so this should probably live in FileDepotMixin.  If you need the action name to be unique you could always do substitutions with `self.display_name` in an i18n suitable way.

Reply:
https://github.com/ManageIQ/manageiq/blob/f38aa50ad118fad5ce95434af76ee1eced0eddb9/app/models/mixins/file_depot_mixin.rb#L19